File: SOLUTIONS/FizzBuzz.cpp
#include <iostream>
using namespace std;
int main() {
	int n, t;
	cin >> t;
	while(t--) {
		cin >> n;
		for (int i = 1; i <= n; ++i)
		{
			if((i%3 != 0) && (i%5 != 0)) cout << i;
			if(i%3 == 0) cout << "Fizz";
			if(i%5 == 0) cout << "Buzz";
			cout << endl;
		}
	}
	return 0;
}

File: SOLUTIONS/FizzBuzz.py
r1=int(input("Please enter the starting number: "))
r2=int(input("Please enter the ending number: "))
for i in range(r1,r2+1):
  if(i%3==0 and i%5!=0):
    print("Fizz")
  elif(i%3!=0 and i%5==0):
    print("Buzz")
  elif(i%3==0 and i%5==0):
    print("FizzBuzz")
  else:
    print(i)
